docs/user/installation_and_configuration.rst

```diff
@@ -132,26 +132,23 @@ For Accumulo 1.6+
 
 Copying the runtime JAR to each tablet server as for Accumulo 1.5 above will
 still work, but in Accumulo 1.6, we can leverage namespaces to isolate the
-GeoMesa classpath from the rest of Accumulo. First, you have to create the
-namespace in the Accumulo shell:
+GeoMesa classpath from the rest of Accumulo.
 
-.. code::
+Execute the ``install-geomesa-namespace.sh`` script in the ``geomesa-$VERSION/dist/accumulo`` directory.
 
-    $ accumulo shell -u root
-    > createnamespace myNamespace
-    > grant NameSpace.CREATE_TABLE -ns myNamespace -u myUser
-    > config -s general.vfs.context.classpath.myNamespace=hdfs://NAME_NODE_FDQN:54310/accumulo/classpath/myNamespace/[^.].*.jar
-    > config -ns myNamespace -s table.classpath.context=myNamespace
+.. code::
```

Comment:
I would not remove the instructions for configuring the JAR in the namespace manipulation, even if the script takes care of it. Maybe we could put all of it into an admonition box or move it to a section in the Accumulo section?